releasenotes/notes/deprecate-old-qpy-d39c754d82655400.yaml

```diff
@@ -0,0 +1,6 @@
+---
+deprecations:
+  - |
+    The import ``qiskit.circuit.qpy_serialization`` is deprecated, as QPY has been promoted to the
+    top level.  You should import the same objects from :mod:`qiskit.qpy` instead.  The old path
+    will be removed in a future of Qiskit Terra.
```

Comment:
I'd probably reword this a bit because "top level" might not be clear to people. But we can bikeshed over that in the release prep PR.